jstests/telemetry/telemetry_auth.js

```diff
@@ -146,13 +146,7 @@ for (const getTelemetryFunc of [getMongodTelemetry, getMongosTelemetry]) {
     testAuthMethodFields(getTelemetryFunc,
                          "GSSAPI,MONGODB-X509,PLAIN",
                          ["kerberos_enabled", "x509_enabled", "ldap_sasl_authentication_enabled"]);
-
-    // check if OIDC feature is enabled
-    if (defaultData.pro_features.includes('OIDC')) {
-        testAuthMethodFields(getTelemetryFunc, "MONGODB-OIDC", "oidc_enabled", oidcOptions);
-    } else {
-        print("OIDC feature is not enabled, skipping oidc_enabled test");
-    }
+    testAuthMethodFields(getTelemetryFunc, "MONGODB-OIDC", "oidc_enabled", oidcOptions);
```

Comment:
It looks like we can still build without some of the features including the OIDC. In such case I would still expect the telemetry suite to pass.